CHANGES.rst

```diff
@@ -69,6 +69,9 @@
 - Add ``AsdfSearchResult.replace`` method for assigning new values to
   search results. [#981]
 
+- Search for block index starting from end of file. Fixes rare bug when
+  a data block contains a block index. [#989]
```

Comment:
These are PR numbers and not issue numbers, so this just needs to be bumped up to `#990`